src/text/helper.ts

```diff
@@ -77,6 +77,18 @@ const getFontWeight = (weight: TextWeight) => {
 // =============================================================================
 // TEXT STYLE
 // =============================================================================
+const getMaxLinesLineStyle = (maxLines: number | undefined = undefined) => {
+    if (maxLines > 0) {
+        return css`
+            display: -webkit-box !important;
+            -webkit-box-orient: vertical;
+            -webkit-line-clamp: ${maxLines};
+            line-clamp: ${maxLines};
```

Comment:
I think `line-clamp` is not an official css property yet, is this more for future compatibility?

Reply:
yes, pls advice remain or remove

Reply:
I noticed the styled components plugin recognises this property so it's fine